kolibri/utils/options.py

```diff
@@ -32,6 +32,28 @@
             "envvars": ("KOLIBRI_DATABASE_HOST",),
         },
     },
+    "Server": {
+        "SERVER_THREAD_POOL": {
+            "type": "integer",
+            "default": 10,
+            "envvars": ("KOLIBRI_SERVER_THREAD_POOL",),
+        },
```

Comment:
Not sure if a previous comment of mine has been lost during force-pushing or never made it. Anyways, I would suggest renaming `SERVER_THREAD_POOL` to `CHERRYPY_THREAD_POOL` and the envvar to `KOLIBRI_CHERRYPY_THREAD_POOL`.